tests/sentry/rules/processing/test_delayed_processing.py

```diff
@@ -54,6 +57,39 @@ def _callthrough_with_order(*args, **kwargs):
     return _callthrough_with_order
 
 
+@freeze_time(FROZEN_TIME)
+class CreateEventTestCase(TestCase, BaseEventFrequencyPercentTest):
```

Comment:
This needs to inherit from `BaseEventFrequencyPercentTest` in order to use the correct `create_event` which doesn't store created events btwn test runs